src/pages/home/CategoryCard.jsx

```diff
@@ -0,0 +1,15 @@
+import { useNavigate } from "react-router-dom";
+
+export function CategoryCard({value}){
+    const {categoryName,image} = value;
+    const navigate = useNavigate();
+
+    return (
+        <div 
+        className="category-card" 
+        onClick={()=>navigate("/explore")}>
+            <img src={image.src} alt={image.alt} loading="lazy" />
```

Comment:
Appreciation : Like how you have added loading='lazy' for the image.

Reply:
Thank you